spring-data-geode-test/src/main/java/org/springframework/data/gemfire/tests/integration/ForkingClientServerIntegrationTestsSupport.java

```diff
@@ -53,70 +51,75 @@
 @SuppressWarnings("unused")
 public abstract class ForkingClientServerIntegrationTestsSupport extends ClientServerIntegrationTestsSupport {
 
-	private static ProcessWrapper gemfireServer;
+    private static List<ProcessWrapper> processes = new LinkedList<>();
+
+    public static void startGemFireServer(Class<?> gemfireServerConfigurationClass, String... arguments)
+            throws IOException {
 
-	public static void startGemFireServer(Class<?> gemfireServerConfigurationClass, String... arguments)
-			throws IOException {
+        int availablePort = setAndGetPoolPortProperty(setAndGetCacheServerPortProperty(findAvailablePort()));
```

Comment:
Please configure your IDE to avoid spacing changes, which have nothing to do with the change set related to this PR.